google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -456,11 +456,21 @@ TEST_F(DataTypeIntegrationTest, JsonMaxNesting) {
     // The emulator has no such limitation, so it tries to re-insert.
     EXPECT_THAT(result, StatusIs(StatusCode::kAlreadyExists));
   } else {
+#if 1
+    // NOTE: The service is currently (August 2022) in a state where there
+    // is no workable "spanner_json_max_nesting_level".  It accepts 80, but
+    // crashes on some yet-to-be discovered value in [82..90].  81 results
+    // in ALREADY_EXISTS.  We really want to get out of the "JsonMaxNesting"
```

Comment:
Let's do that.  Set the constant to 20.  Or remove this test altogether.  The teams that own this stuff should write their own tests.

Reply:
I'm totally on board with that (removing the test altogether).  But I propose that we wait until the current issue is addressed.

Reply:
SGTM